src/mlpack/core/dists/gamma_distribution.hpp

```diff
@@ -171,6 +172,14 @@ class GammaDistribution
      */
     void LogProbability(const arma::mat& observations,
                         arma::vec& LogProbabilities) const;
+    /*
```

Comment:
Please include a newline here. Also another `*` I think.

Reply:
@ShikharJ I have included the changes.